lib/benchee/profile.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Benchee.Profile do
+  @moduledoc """
+  Profiles each scenario after benchmarking them if the `profile_after` flag is set to `true`.
+
+  The profiler that will be used is either the one set by the `profiler` option
+  in the `profile` configuration or the default one (`:cprof`). It accepts however the following
+  profiler options:
+    * `:cprof` will profile with `Mix.Task.Profile.Cprof` (useful to discover bottlenecks
+    related to function calls).
+    * `:eprof` will profile with `Mix.Task.Profile.Eprof` (useful to discover bottlenecks
+    related to time information of each function call).
+    * `:fprof` will profile with `Mix.Task.Profile.Fprof` (useful to discover bottlenecks
+    of a sequential code).
```

Comment:
what does it mean of sequential code? :thinking:  Like, non parallel code? Unsure what that tells me :)

Reply:
I took it from the Elixir documentation, but I've updated all three descriptions to be a bit more descriptive :)